packages/design-system/src/components/utilities/typographyHeadings.stories.tsx

```diff
@@ -25,6 +25,23 @@ export const AllHeadings = () => {
     </>
   );
 };
+export const AllHeadingsOnDark = () => {
```

Comment:
Same thing can be done to this function

Reply:
Thanks for pointing out and suggesting all these great updates @pwolfert :partying\_face: